ocs_ci/utility/aws.py

```diff
@@ -427,6 +458,159 @@ def get_ec2_instance_volumes(self, instance_id):
+            all_sg_ids.append(security_group_id)
+            ec2_instance.modify_attribute(Groups=all_sg_ids)
+
+        logger.info(f"Security Group {security_group_id} added to selected node")
+
+    def remove_security_group(self, security_group_id, instance_id):
+        """
+        Remove security group from selected ec2 instance (by instance id)
+        print out: security group <id> removed from selected nodes
+
+        Args:
+            security_group_id (str): Security group to be removed
+            instance_id (str): Instance attached with selected security group
+
+        """
+        ec2_instance = self.get_ec2_instance(instance_id)
+        logger.info(f"ec2_instance id is: {ec2_instance.id}")
+        all_sg_ids = self.get_security_groups_by_instance_id(instance_id)
+        for sg in all_sg_ids:
+            if sg == security_group_id:
+                all_sg_ids.remove(security_group_id)
+                ec2_instance.modify_attribute(Groups=all_sg_ids)
+                logger.info(f"Security Group {security_group_id} removed from selected node")
+
+    def delete_security_group(self, security_group_id):
+        """
+        Delete selected security group
+        print out: Security group <id> deleted
+
+        Args:
+            security_group_id (str): Id of selected security group
+
+        """
+        self.ec2_client.delete_security_group(GroupId=security_group_id)
+        logger.info(f"Security group {security_group_id} deleted")
+
+    def store_security_groups_for_instances(self, instances_id):
+        """
+        Stored all security groups attached to selected ec2 instances
+
+        Args:
+            instances_id (list): ec2 instance_id
+
+        Returns:
+            dict: security_group_dict: keys: blocked instances: ec2_instances ids
+                values: list of original security groups of each instance
+
+        """
+        sg_list = list()
+        for instance in instances_id:
+            sg_list.append(self.get_security_groups_by_instance_id(instance))
+
+        return dict(zip(instances_id, sg_list))
+
+    def block_instances_access(self, security_group_id, instances_id):
+        """
+        Block ec2 instances by:
+
+        - Append security group without access permissions
+        - Remove original security groups
+
+        Args:
+            security_group_id (str): security group without access permissions
+            instances_id (list): list of ec2 instances ids
+
+        """
+
+        for instance in instances_id:
+            original_sgs = self.get_security_groups_by_instance_id(instance)
+            self.append_security_group(security_group_id, instance)
+            logger.info(f"APPEND, INSTANCE {instance}, SEC_GRP_ID: {security_group_id}")
+            for sg_grp in original_sgs:
+                self.remove_security_group(sg_grp, instance)
+                logger.info(f"REMOVE, INSTANCE {instance}, SEC_GRP_ID: {sg_grp}")
```

Comment:
no need for capital letters, please fix

Reply:
the capital letters was for my debugging, for some reason i forgot to remove them